static/style/chart.css

```
@@ -0,0 +1,121 @@
+.chart-container {
+  max-width: 15vw;
```

This makes it scale with the screen size, but the content column (`<main>`) doesn't grow beyond a certain point. 20% of that looks about right to my eye. 

But that's positively minuscule on a phone, so let's go with a fixed pixel size.

```suggestion
  max-width: 260px;
```

Unless you want to go ahead and pad the data up to a full 52 weeks, so the chart doesn't grow in size every time we have new data?